huaweicloud/services/acceptance/apig/resource_huaweicloud_apig_instance_test.go

```diff
@@ -69,6 +69,7 @@ func TestAccInstance_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "ingress_bandwidth_size", "0"),
 					resource.TestCheckResourceAttr(resourceName, "egress_address", ""),
 					resource.TestCheckResourceAttr(resourceName, "ingress_address", ""),
+					resource.TestCheckResourceAttr(resourceName, "custom_ingress_ports.#", "2"),
```

Comment:
Suggest to reorder these steps:
1. testing whether one custom ingress port can be created during CreateContext method
2. testing whether another custom ingress port can be appended during UpdateContext method
3. testing whether all custom ingress ports can be deleted during UpdateContext method
4. ......

Reply:
done